src/workerd/io/worker.c++

```diff
@@ -1194,6 +1194,17 @@ Worker::Isolate::Isolate(kj::Own<Api> apiParam,
                   v8::Local<v8::Value> argument)> callback) -> v8::Maybe<void> {
         try {
           auto& js = jsg::Lock::from(isolate);
+
+          // TODO(soon): Hopefully this logging is temporary. We want to get an idea
+          // of where the cross request promises are being resolved just in general.
+          // To do so we need to try to capture a stack. We do so by creating an error
+          // object and logging it.
+          auto err = v8::Exception::Error(js.str(""_kj)).As<v8::Object>();
+          jsg::check(err->Set(
+              js.v8Context(), js.str("name"_kj), js.str("Cross Request Promise Resolve"_kj)));
```

Comment:
Is setting the name actually important? Couldn't we put `Cross Request Promise Resolve` (or something else greppable) into the KJ_LOG line below instead?

Reply:
If we don't the log line would say `Error: ...` and I want to avoid folks getting confused about it.

Reply:
OK but currently the log line will be

`NOSENTRY Cross Request Promise Resolve: `

Which is also confusing.